lib/aptible/cli/resource_formatter.rb

```diff
@@ -49,7 +65,9 @@ def inject_account(node, account)
         def inject_operation(node, operation)
           node.value('id', operation.id)
           node.value('status', operation.status)
-          node.value('git_ref', operation.git_ref)
+          unless operation.type == 'backup'
```

Comment:
I'm curious, what was the reason for adding this check?  It seems like `created_from_operation` shouldn't have affected the need for this check.

Reply:
As written in this PR, it does look a bit awkward. We only show operations in two places: the last deploy operation of an App (in `aptible apps`) and here in the backups. So, I exclude showing it for backup operations.

Looking forward to showing all operation types in the CLI, I would expect git_ref to only be relevant (the only places it could be non-nil) in two types of operations. Would following be more or less clear in the context of this PR?

```
if %w(deploy rebuild).include?(op.type)
  node.value('git_ref', op.git_ref)
end
```

Reply:
yes i think so

Reply:
Updated!